libs/features/pages/home/src/lib/main/home/home-page.component.ts

```diff
@@ -1,8 +1,11 @@
 import { Component } from '@angular/core';
+import { ImageConfig } from '@elewa-website/models/schema/ui/images';
 
 @Component({
   selector: 'elewa-website-home',
   templateUrl: './home-page.component.html',
   styleUrls: ['./home-page.component.scss'],
 })
-export class HomePageComponent {}
+export class HomePageComponent {
+  public imageconfigs: ImageConfig[] = []
+}
```

Comment:
No need to push these changes as our HomePage component does not need this import. Any Component that needs this will import it.